core/src/main/java/net/opentsdb/query/execution/serdes/JsonV3QuerySerdes.java

```diff
@@ -102,6 +109,12 @@
   //<set ID, <ts hash, ts wrapper>>
   private Map<String, Map<Long, SeriesWrapper>> partials = Maps.newConcurrentMap();
   private List<byte[]> serialized_results = Lists.newArrayList();
+  
+  /** Used for the parallel stream*/
+  private ForkJoinPool customThreadPool = new ForkJoinPool(8);
```

Comment:
need to static it so we don't create it each time.

Reply:
good catch